### DISCUSSION TITLE
Prioritize Chroma?

### DISCUSSION THREAD

**Discussion by hamaryns** [beetbox/beets#5493] [Q&A]
Hi, I seem to have seen an example of a config file once where someone had a setting to change the priority with which beets uses information for audiotagging.  I want Chroma to have higher priority than metadata.  Can anyone point me to the right setting?
TIA, H.

**Comment by jackwilsdon** [comment]
It sounds like you're looking for `match.distance_weights.track_id`: https://github.com/beetbox/beets/blob/fa10dcf11add0afd3b4b22af29f8d504e7ef8a0a/beets/config_default.yaml#L209

This sets how important the track ID is when sorting matches (it's worth noting track ID is provided both by Chroma and also the embedded MBID tag in files - there's no way to disable this at the moment).

I found this configuration on the forums which looks like it's attempting to prioritize Chroma, not sure if it's the one you're thinking of: https://discourse.beets.io/t/chroma-match-information-is-dropped-not-available-for-musicbrainz/1059/20?u=jackwilsdon
